section1/reflection.md

```diff
@@ -2,18 +2,50 @@
 
 1. Regarding the blog posts in Part A, how do you feel about asking questions? Do you tend to ask them too soon, or wait too long, or somewhere in between?
 
+>My typical way to ask a question is to first research what I can and then try to ask the question and include what I think is the answer.  I usually do think to ensure I am on the right track before committing but I also think it can be a way to avoid looking foolish asking a "dumb question".  I do not believe I am bad at asking questions I think I struggle more with constructing them.
```

Comment:
I appreciate this - and want to reinforce that writing up the question itself IS VERY DIFFICULT - I think you are doing it right if you struggle a bit with that.